ldap/src/integration-test/java/org/springframework/security/ldap/ApacheDsContainerConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.ldap;
+
+import javax.annotation.PreDestroy;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.ldap.core.ContextSource;
+import org.springframework.security.ldap.server.ApacheDSContainer;
+
+/**
+ * @author Edd� Mel�ndez
+ */
+@Configuration
+public class ApacheDsContainerConfig {
+
+	ApacheDSContainer container = new ApacheDSContainer("dc=springframework,dc=org",
+			"classpath:test-server.ldif");
+
+	public ApacheDsContainerConfig() throws Exception {
```

Comment:
Is this constructor necessary and could you help me understand why, if so?

Reply:
I made a change here so this is not needed anymore.